content/it/continuous-delivery(CD).md

```diff
@@ -0,0 +1,14 @@
+---
+title: Continuous Delivery (CD)
+status: Completed
+category: Concetto
+---
+
+## Cos'è
+La consegna continua, spesso abbreviata in CD, è un insieme di pratiche in cui le modifiche al codice vengono automaticamente implementate in un ambiente di accettazione (oppure, nel caso di distribuzione continua, in produzione). Il CD include in modo fondamentale le procedure per garantire che il software sia adeguatamente testato prima della distribuzione e fornisce un modo per ripristinare le modifiche se ritenuto necessario. L'integrazione continua (CI) è il primo passo verso la distribuzione continua (ovvero, le modifiche devono essere unite in modo pulito prima di essere testate e implementate).
```

Comment:
Vedi la notazione sui termini intraducibili https://docs.google.com/document/d/1g4__gy--tB79UH_KMcWFNbJpPQQUhlVpqnKKGGndOZU/edit#bookmark=id.bxndpdwsldp0
```suggestion
Il _Continuous Delivery_ (consegna continua), spesso abbreviato in CD, è un insieme di pratiche in cui le modifiche al codice vengono automaticamente implementate in un ambiente di accettazione (oppure, nel caso di distribuzione continua, in produzione). Parte integrante del CD sono le procedure per garantire che il software sia adeguatamente testato prima della distribuzione, nonché le modalità di ripristino in caso di necessità. L'integrazione continua (CI - da _Continuous Integration_) è il primo passo verso la distribuzione continua (vale a dire, le modifiche devono essere unite in modo pulito prima di essere testate e implementate).
```

Reply:
- propongo _**La**_ Continuous Delivery (quindi poi tutto deve essere declinato al femminile)
- `(vale a dire, le modifiche devono essere unite in modo pulito prima di essere testate e implementate).` unite? so che non possiamo usare mergiate 😝 - magari _raggruppate_?

Reply:
fatto nel suggerimento. Tu si che trovi le parole giuste!

Reply:
deh! non prendermi in giro, sa'! più che altro, sono molto indecisa sul termine "consegna" per delivery...

Reply:
@meryem-ldn @sistella ci ho pensato, ma "consegna continua" proprio non va, a mio parere. propongo "rilascio continuo in ambiente di test". in italiano, in ogni caso, rimane Continuous Delivery, se anche la traduzione risulta una lunga locuzione non sconvolge nessuno. fatemi sapere :)

Reply:
Questo è proprio il senso della frase: tieni presente che qui si è voluto rendere con `distribuzione continua` il termine Continuous Deployment (https://glossary.cncf.io/continuous_deployment/), che differisce appunto nel fatto che il processo è 'stirato' fino a produzione. 
![blablabla](https://media.giphy.com/media/jnhXd7KT8UTk5WIgiV/giphy.gif)

Reply:
AHHAHHAHHHHAH. bom, me ne torno buona al banco.